test/openAPI/features/support/helpers/helpers.js

```diff
@@ -369,4 +369,24 @@ module.exports = {
     },
     required: ['ID', 'FirstName', 'LastName', 'BirthCertificateID'],
   },
+  replaceKeyWithValueFromJson,
+  readJsonFile,
 };
+
+const fs = require('fs');
+let jsonData = readJsonFile('testCustomParameters.json') || readJsonFile('testDefaultParameters.json');
```

Comment:
I think that jsonData should be const. Also I'm not sure about this files. Do we have any validation to confirm that custom parameters match schema of default parameters?

Reply:
Yes we do, we have custom_param_manager.sh that compares keys from both of the dictionaries. It assume that the default one is properly created, which can be checked by running tests, because if any key is missing then tests will raise an error.